src/configs/settings.ts

```diff
@@ -1,6 +1,20 @@
 /** This is the main configuration module */
 import { Providers } from '@onaio/gatekeeper';
 import { Expression } from 'mapbox-gl';
+import {
```

Comment:
Please dont do this import.  I'm fine with having the string literal here.  We dont want settings to depend on anything in the codebase